From 8393d5f3937bc21964553e716b9f4e90ea3ce75c Mon Sep 17 00:00:00 2001
Message-Id: <8393d5f3937bc21964553e716b9f4e90ea3ce75c.1300791642.git.jdenemar@redhat.com>
From: Eric Blake <eblake@redhat.com>
Date: Fri, 18 Mar 2011 12:02:32 -0600
Subject: [PATCH] add additional event debug points

https://bugzilla.redhat.com/show_bug.cgi?id=624252

Followup to commit 2222bd24
(cherry picked from commit 6a8ef183bef1fec74590bfde315da9ad0f5007ac)

Conflicts:

	src/remote/remote_driver.c - context changed s/DEBUG/VIR_&/
---
 src/remote/remote_driver.c |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/src/remote/remote_driver.c b/src/remote/remote_driver.c
index 0c65068..a7265b5 100644
--- a/src/remote/remote_driver.c
+++ b/src/remote/remote_driver.c
@@ -9958,7 +9958,6 @@ processCallDispatchMessage(virConnectPtr conn, struct private_data *priv,
     /* An async message has come in while we were waiting for the
      * response. Process it to pull it off the wire, and try again
      */
-    DEBUG0("Encountered an event while waiting for a response");
 
     if (in_open) {
         DEBUG("Ignoring bogus event %d received while in open", hdr->proc);
@@ -9998,6 +9997,7 @@ processCallDispatchMessage(virConnectPtr conn, struct private_data *priv,
         DEBUG("Unexpected event proc %d", hdr->proc);
         break;
     }
+    VIR_DEBUG("Event ready for queue %p %p", event, conn);
 
     if (!event)
         return -1;
@@ -10651,6 +10651,7 @@ static void remoteDomainEventDispatchFunc(virConnectPtr conn,
 
     /* Drop the lock whle dispatching, for sake of re-entrancy */
     remoteDriverUnlock(priv);
+    VIR_DEBUG("Dispatch event %p %p", event, conn);
     virDomainEventDispatchDefaultFunc(conn, event, cb, cbopaque, NULL);
     remoteDriverLock(priv);
 }
@@ -10664,6 +10665,7 @@ remoteDomainEventQueueFlush(int timer ATTRIBUTE_UNUSED, void *opaque)
 
     remoteDriverLock(priv);
 
+    VIR_DEBUG("Event queue flush %p", conn);
     priv->domainEventDispatching = 1;
 
     /* Copy the queue, so we're reentrant safe */
-- 
1.7.4.1

